packages/types/src/utils.ts

```diff
@@ -1,17 +1,25 @@
-import { PaymentTypes } from '.';
+import { ExtensionTypes } from '.';
```

Comment:
```suggestion
import { ExtensionTypes } from './extension-types';
```

(avoids potential circular imports)